api/http/handler/edgejobtasks/handler.go

```diff
@@ -0,0 +1,35 @@
+package edgejobtasks
```

Comment:
Why not integrating the `edgejobtasks` resource in the same handler as `edgejob`? I'd rather keep all the resources/sub-resources inside the same package.